### PR TITLE
intro block fix after testing

### PR DESCRIPTION
**Pull request by adriKosa**

### CODE REVIEW

File: src/idsk/components/search-component/search-component.scss [hunk 1]
```diff
@@ -69,8 +66,8 @@
     z-index: 2;
 
     svg {
-      height: 22px;
-      width: 18px;
+      height: 18,14px;
+      width: 18,14px;
     }
```
Comment (viloDlugos) [inline src/idsk/components/search-component/search-component.scss:71]: tu nemoze byt height aj weight : 18px ?

File: src/idsk/components/search-component/template.njk [hunk 2]
```diff
@@ -6,7 +6,7 @@
     <button
       type="submit"
       class="govuk-button idsk-search-component__button {{ 'idsk-search-component__button--small' if params.isSmall }}">
-      <svg width="31" height="30" viewbox="0 0 31 30" fill="none" xmlns="http://www.w3.org/2000/svg">
+      <svg width="18,14" height="18,14" viewbox="0 0 31 30" fill="none" xmlns="http://www.w3.org/2000/svg">
         <path
```
Comment (viloDlugos) [inline src/idsk/components/search-component/template.njk:10]: tu tiez nemoze byt 18 ?

File: src/idsk/components/search-component/search-component.scss (hunk 1 above)
Comment (adriKosa) [reply src/idsk/components/search-component/search-component.scss:71]: moze

File: src/idsk/components/search-component/template.njk (hunk 2 above)
Comment (adriKosa) [reply src/idsk/components/search-component/template.njk:10]: tiez moze

**Review by viloDlugos — APPROVED** [review]

File: app/views/komponenty/intro-block.njk [hunk 3]
```diff
@@ -16,7 +16,7 @@
     <span class="govuk-caption-xl">Webové komponenty</span>
     <h1 class="govuk-heading-xl">{{pageName}}</h1>
     <p class="govuk-body-l">
-        Pri tvorbe novej webovej stránky je okrem povinnej hlavičky pre webové sídla [link na hlavičku pre webové sídla] a pätičky s feedback barom [link na pätičku pre webové sídla] povinný aj tento úvodný blok.
+        Pri tvorbe novej webovej stránky je okrem povinnej <a href="/komponenty/hlavicka-webove-sidla" target="_blank" title="hlavičky pre webové sídla">hlavičky pre webové sídla</a> a <a href="/komponenty/footer-extended" target="_blank" title="pätičky s feedback barom">pätičky s feedback barom</a> povinný aj tento úvodný blok.
```
Comment (LubosKat) [inline app/views/komponenty/intro-block.njk:19]: govuk-link

File: src/idsk/components/intro-block/template.njk [hunk 4]
```diff
@@ -3,9 +3,9 @@
         <div class="govuk-grid-row {{ params.classContainer }}">
             <div class="govuk-grid-column-full govuk-grid-column-two-thirds-from-desktop">
                 <h2 class="govuk-heading-m">
-                    {{ params.headline if params.headline else "Ministerstvo investícií, regionálneho rozvoja a informatizácie Slovenskej republiky"}}
+                    {{ params.headline if params.headline else "Vitajte na MIRRI SR"}}
                 </h2>
-                <p class="govuk-body">{{ params.mainTitle if params.mainTitle else "Vitajte na stránke Ministerstva investícií, regionálneho rozvoja a informatizácie. Dozvedieť sa"}}
+                <p class="govuk-body">{% if params.mainTitle %} params.mainTitle {% else %} Dozvedieť sa <a class='govuk-link' href='#' title='viac o ministerstve'>viac o ministerstve</a>. {% endif %}
                     {% if params.mainTitleLinkLabel %}
                     <a class="govuk-link" href="{{ params.moreInfoAboutMinistryLink if params.moreInfoAboutMinistryLink else '#'  }}">
```
Comment (LubosKat) [inline src/idsk/components/intro-block/template.njk:10]: title attribute

**Pull request merged**